src/stories/autosuggest-text/AutosuggestText.stories.tsx

```diff
@@ -0,0 +1,54 @@
+import { ComponentMeta, ComponentStory } from "@storybook/react";
+import { withDesign } from "storybook-addon-designs";
+import { AutosuggestText } from "./AutosuggestText";
+
+export default {
+  title: "Components / Autosuggest - Text",
+  component: AutosuggestText,
+  decorators: [withDesign],
+  argTypes: {
+    items: {
+      name: "Items",
+      defaultValue: ["Item one", "Item two"],
+      control: { type: "array" }
+    }
+  },
+  parameters: {},
+} as ComponentMeta<typeof AutosuggestText>;
+
+const Template: ComponentStory<typeof AutosuggestText> = (args) => (
+  // apart from SuggestText, everything else is here just for the story context
+  // SuggestText style is directly dependent on the header and search field styling
+  <>
+    <header className="header" style={{height: "144px"}}>
```

Comment:
@spaceo because it doesn't hold its height without it. It would probably need all 1 million of its children elements, which I wanted to avoid. The story context just mimics the real component, right?